### PR TITLE
Update demos

### PR DESCRIPTION
**Pull request by zorkow**
* Updates all demos to work with the latest release of MathJax.
* Uniformly allows to supply additional configuration parameters for SRE.
* Adds four new scripts:
     *  `tex2svg-page`, self-explanatory
     * `tex2chtml-speech`, single formula translation using the `tex-chtml-full-speech` component
     * `tex2chtml-speech`, page translation using the `tex-chtml-full-speech` component
     * `mml2mml`, single formula translation from MathML to MathML with `alttext` attribute

### CODE REVIEW

File: speech/action.js [hunk 1]
```diff
@@ -45,7 +93,7 @@ exports.sreconfig = function(data) {
 //    The second is for when a MathItem's render(), rerender() or convert() method is called.
 //
 exports.speechAction = {
-  simplfy: [
+  simplify: [
     STATE.ENRICHED + 1,
```
Comment (dpvc) [inline speech/action.js:97]: I'd recommend defining a state for the render actions, for clarity.  Use `newState()` from `MathItem.js` (which you are require already anyway).  E.g.,

``` js
newState('SIMPLIFY', STATE.ENRICHED + 1);
newState('ROLE', 1000);
newState('DESCRIBE', STATE.ROLE + 1);
```

and then use these here and below.

File: speech/mml2mml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+const action = require('./action.js');
+
+//
+// Add a render action to move the computed speech into the alttext attribute.
+//
+function moveSpeech(math) {
+  let alttext = '';
+  math.root.walkTree(node => {
+    const attributes = node.attributes.getAllAttributes();
+    console.log(attributes);
```
Comment (dpvc) [inline speech/mml2mml:64]: Is this `console.log()` for debugging, and should be removed?

File: speech/tex2svg [hunk 3]
```diff
@@ -147,8 +160,8 @@ MathJax.startup.promise.then(() => {
         if (argv.css) {
             console.log(adaptor.textContent(MathJax.svgStylesheet()));
         } else {
-            let html = (argv.container ? adaptor.outerHTML(node) : adaptor.innerHTML(node));
-            console.log(argv.styles ? html.replace(/<defs>/, `<defs><style>${CSS}</style>`) : html);
+          let html = argv.container ? adaptor.outerHTML(node) : adaptor.innerHTML(node);
+            console.log(argv.styles ? html.replace(/<defs>/, `<defs><style aria-hidden="true">${CSS}</style>`) : html);
```
Comment (dpvc) [inline speech/tex2svg:164]: Remove two spaces here, as in the line above?

File: speech/action.js [hunk 4]
```diff
@@ -19,22 +19,70 @@ function removeSemanticData(math) {
 }
 
 
-const sreDefault = {
-    domain: 'mathspeak',
-    style: 'default'
-};
+//
+// Moves all speech elements into aria-labels for SVG output. This allows for
+// elements without containers some limited exploration.
```
Comment (dpvc) [inline speech/action.js:24]: Add "containers **to have** some"?

File: speech/action.js [hunk 5]
```diff
@@ -19,22 +19,70 @@ function removeSemanticData(math) {
...
+//
+// Note, that here we cannot walk the source tree, as we alterations are to be
```
Comment (dpvc) [inline speech/action.js:26]: Remove "we"?

**Review by dpvc — APPROVED** [review]
Some typos in comments, and some small suggestions.  Otherwise OK.

File: speech/mml2mml (hunk 2 above)
Comment (zorkow) [reply speech/mml2mml:64]: Removed.

File: speech/action.js (hunk 1 above)
Comment (zorkow) [reply speech/action.js]: Done. And added more explanations in comments.

**Pull request merged**